src/main/java/io/anserini/search/topicreader/Topics.java

```diff
@@ -45,52 +48,71 @@ public enum Topics {
-  TREC2019_DL_PASSAGE(TsvIntTopicReader.class,"topics-and-qrels/topics.dl19-passage.txt"),
-  TREC2019_DL_PASSAGE_WP(TsvIntTopicReader.class,"topics-and-qrels/topics.dl19-passage.wp.tsv.gz"),
-  TREC2019_DL_PASSAGE_UNICOIL(TsvIntTopicReader.class,"topics-and-qrels/topics.dl19-passage.unicoil.0shot.tsv.gz"),
-  TREC2019_DL_PASSAGE_UNICOIL_NOEXP(TsvIntTopicReader.class,"topics-and-qrels/topics.dl19-passage.unicoil-noexp.0shot.tsv.gz"),
-  TREC2019_DL_PASSAGE_SPLADE_DISTILL_COCODENSER_MEDIUM(TsvIntTopicReader.class,"topics-and-qrels/topics.dl19-passage.splade_distil_cocodenser_medium.tsv.gz"),
-  TREC2020_DL(TsvIntTopicReader.class,"topics-and-qrels/topics.dl20.txt"),
-  TREC2020_DL_WP(TsvIntTopicReader.class,"topics-and-qrels/topics.dl20.wp.tsv.gz"),
-  TREC2020_DL_UNICOIL(TsvIntTopicReader.class,"topics-and-qrels/topics.dl20.unicoil.0shot.tsv.gz"),
-  TREC2020_DL_UNICOIL_NOEXP(TsvIntTopicReader.class,"topics-and-qrels/topics.dl20.unicoil-noexp.0shot.tsv.gz"),
-  TREC2020_DL_SPLADE_DISTILL_COCODENSER_MEDIUM(TsvIntTopicReader.class,"topics-and-qrels/topics.dl20.splade_distil_cocodenser_medium.tsv.gz"),
-  TREC2021_DL(TsvIntTopicReader.class,"topics-and-qrels/topics.dl21.txt"),
-  TREC2021_DL_UNICOIL(TsvIntTopicReader.class,"topics-and-qrels/topics.dl21.unicoil.0shot.tsv.gz"),
-  TREC2021_DL_UNICOIL_NOEXP(TsvIntTopicReader.class,"topics-and-qrels/topics.dl21.unicoil-noexp.0shot.tsv.gz"),
-  TREC2022_DL(TsvIntTopicReader.class,"topics-and-qrels/topics.dl22.txt"),
-  TREC2022_DL_UNICOIL(TsvIntTopicReader.class,"topics-and-qrels/topics.dl22.unicoil.0shot.tsv.gz"),
-  TREC2022_DL_UNICOIL_NOEXP(TsvIntTopicReader.class,"topics-and-qrels/topics.dl22.unicoil-noexp.0shot.tsv.gz"),
-  MSMARCO_DOC_DEV(TsvIntTopicReader.class,"topics-and-qrels/topics.msmarco-doc.dev.txt"),
-  MSMARCO_DOC_DEV_WP(TsvIntTopicReader.class,"topics-and-qrels/topics.msmarco-doc.dev.wp.tsv.gz"),
-  MSMARCO_DOC_DEV_UNICOIL(TsvIntTopicReader.class,"topics-and-qrels/topics.msmarco-doc.dev.unicoil.tsv.gz"),
-  MSMARCO_DOC_DEV_UNICOIL_NOEXP(TsvIntTopicReader.class,"topics-and-qrels/topics.msmarco-doc.dev.unicoil-noexp.tsv.gz"),
-  MSMARCO_DOC_TEST(TsvIntTopicReader.class,"topics-and-qrels/topics.msmarco-doc.test.txt"),
+  TREC2019_DL_DOC(TsvIntTopicReader.class, "topics-and-qrels/topics.dl19-doc.txt"),
+  TREC2019_DL_DOC_WP(TsvIntTopicReader.class, "topics-and-qrels/topics.dl19-doc.wp.tsv.gz"),
+  TREC2019_DL_DOC_UNICOIL(TsvIntTopicReader.class, "topics-and-qrels/topics.dl19-doc.unicoil.0shot.tsv.gz"),
+  TREC2019_DL_DOC_UNICOIL_NOEXP(TsvIntTopicReader.class, "topics-and-qrels/topics.dl19-doc.unicoil-noexp.0shot.tsv.gz"),
+  TREC2019_DL_PASSAGE(TsvIntTopicReader.class, "topics-and-qrels/topics.dl19-passage.txt"),
+  TREC2019_DL_PASSAGE_WP(TsvIntTopicReader.class, "topics-and-qrels/topics.dl19-passage.wp.tsv.gz"),
+  TREC2019_DL_PASSAGE_UNICOIL(TsvIntTopicReader.class, "topics-and-qrels/topics.dl19-passage.unicoil.0shot.tsv.gz"),
+  TREC2019_DL_PASSAGE_UNICOIL_NOEXP(TsvIntTopicReader.class,
```

Comment:
Can we change this back to a single line? I would prefer overlong lines rather than this awkwardness.
Also, we don't need `topics-and-qrels/` right?

Reply:
yeah. we don't need topics-and-qrels